governance.md

```diff
@@ -11,52 +11,46 @@ InstructLab is made up of several projects that are defined as codebases and ser
-
-Project Maintainers are responsible for activities surrounding the development and release of code, the operation of any services that they own, or the tasks needed to execute their project (for example, community management or setting up an event booth). Technical decisions for code resides with the project Maintainers unless there is a decision related to cross maintainer groups that cannot be resolved by those groups. Those cases can be escalated to the [organization Maintainers](https://github.com/instructlab/community/blob/main/MAINTAINERS.md).
-
-In some cases, groups of maintainers are responsible for more than one repository, such as `ilab` CLI tool maintainers managing the [taxonomy repository](https://github.com/instructlab/taxonomy).
+Project Maintainers focus on a single codebase, a group of related codebases, a service (for example, a website), or a project to support other projects (such as marketing or community management).
 
-To be considered an _active project Maintainer_, it is required to be associated with at least one active, non-archived project. If only listed on archived projects, they become _emeritus Maintainers_ and are no longer eligible to become an organization Maintainer.
+Project Maintainers are responsible for activities surrounding the development and release of code, the operation of any services that they own, or the tasks needed to execute their project (for example, community management or setting up an event booth). Technical decisions for code reside with the project Maintainers unless there is a decision related to multiple maintainer groups that cannot be resolved by those groups. Those cases can be escalated to the Oversight Committee.
 
-Project Maintainers do not need to be software developers; however they must be substantial contributors. For example, if a repository is for documentation it would be appropriate for a project Maintainer to be an editor or technical writer.
+Project Maintainers do not need to be software developers; however, they must be substantial contributors. For example, if a repository is for documentation it would be appropriate for a project Maintainer to be an editor or technical writer.
 
 Advancement to the project Maintainer position, removal or stepping down, and duties are detailed in the [Contributor Roles](https://github.com/instructlab/community/blob/main/CONTRIBUTOR_ROLES.md). The list of current maintainers can be found [here](https://github.com/instructlab/community/blob/main/MAINTAINERS.md).
 
-## InstructLab Oversight Committee overview
+## InstructLab Oversight Committee Overview
 
-The Oversight Committee is selected by project Maintainers and consists of 3 to 9 leaders on the InstructLab project; these members will serve to supervise the overall project and its health. It will also consist of a selected chair member who will set agendas and call meetings; these meetings can be public or private at the discretion of the Oversight Committee.
```

Comment:
I do not think that this paragraph needs to be deleted and, in fact, I would urge us to keep it.

Reply:
oh its readded below written a bit differently isnt it